Server/datalayer/user.datalayer.js

```diff
@@ -8,6 +8,49 @@ class usersDatalayer {
     * Initializes the datalayer
     */
 
+    // Creates new user 
+    static createUser(reqbody) {
+	let user = new userModel(
+        {
+            username: reqbody.username,
+            email: reqbody.email,
+            password: reqbody.password,
+            firstname: reqbody.firstname,
+            lastname: reqbody.lastname
+        });
+
+        return user.save().catch(err => {
+        throw new error(`Not able to save - ${err}`);
+      });
+    }
+
+    // Get user details based on id
+    static getUsersByUserId(userid) {
```

Comment:
Use `user` and not `users` as we are finding one user here

Reply:
Done